src/app/components/navbar/navbar.component.html

```diff
@@ -24,13 +24,32 @@
     </ng-container>
 
     <div class="toolbar-spacer"></div>
+
     <div [hidden]="(currentExperiment|async) == null">
         <span>{{currentExperiment|async}}</span>
         <a class="navigation-link" mat-icon-button [hidden]="(currentExperiment|async) == null" [routerLink]="['/']"
             aria-label="change experiment">
             <mat-icon>swap_horiz</mat-icon>
         </a>
     </div>
+    <button class="downloads-button" mat-button [matMenuTriggerFor]="downloadList">
+        <mat-icon [matBadge]="downloadBadgeCounter | async" [matBadgeHidden]="!(downloadBadgeCounter | async)"
+            matBadgeColor="warn" matBadgeSize="small" (click)="resetBadge()">download</mat-icon>
+    </button>
+    <mat-menu class="download" #downloadList="matMenu" yPosition="below" xPosition="before">
+        <mat-card class="card download-card" *ngFor="let export of exportList | async; trackBy:trackExport">
+            <mat-card-content>
+                <ng-container class="container-download">
```

Comment:
If it has a css class it should use a div tag. `ng-container` is for cases where you need nesting in the template but do not need the extra tag in the dom.